tests/datatypes/test_dict.py

```diff
@@ -229,6 +229,66 @@ def test_method_get(self):
                 print(err)
             """)
 
+    def test_copy(self):
+        self.assertCodeExecution("""
+            x = {42: 'Babel'}
+            y = x.copy()
+            print(y)
+            """)
+
+        self.assertCodeExecution("""
+            x = {42: 'Babel'}
+            y = x.copy()
+            print(x == y)
+            """)
+
+        self.assertCodeExecution("""
+            x = {42: 'Babel'}
+            y = x.copy()
+            print(x is not y)
+            """)
+
+        self.assertCodeExecution("""
+            x = {42: 'Babel'}
+            y = x.copy()
+            y['42'] = 'Babel'
```

Comment:
I think it'd useful to modify this test (or add a new test) to change the value for an existing key in `x`, then check that the change is not present in `y`, and that the dicts aren't the same.

This could be followed by another line to similarly test adding a new key to one of the dicts.